keanu-python/keanu/algorithm/optimization.py

```diff
@@ -1,27 +1,45 @@
 from py4j.java_gateway import java_import, JavaObject, JavaClass
+from typing import Union, Optional, Tuple
+
+from keanu.base import JavaObjectWrapper
 from keanu.context import KeanuContext
 from keanu.net import BayesNet
+from keanu.tensor import Tensor
+from keanu.vartypes import numpy_types
 from keanu.vertex.base import Vertex
-from typing import Union, Optional, Tuple
 
 k = KeanuContext()
 
 java_import(k.jvm_view(), "io.improbable.keanu.algorithms.variational.optimizer.gradient.GradientOptimizer")
+java_import(k.jvm_view(), "io.improbable.keanu.algorithms.variational.optimizer.gradient.ConjugateGradient")
 java_import(k.jvm_view(), "io.improbable.keanu.algorithms.variational.optimizer.nongradient.NonGradientOptimizer")
+java_import(k.jvm_view(), "io.improbable.keanu.algorithms.variational.optimizer.nongradient.BOBYQA")
 java_import(k.jvm_view(), "io.improbable.keanu.algorithms.variational.optimizer.KeanuOptimizer")
 
 
+class OptimizedResult(JavaObjectWrapper):
+
+    def __init__(self, result_object: JavaObject):
+        super().__init__(result_object)
+
+    def optimized_fitness(self) -> float:
```

Comment:
From a usability point of view this feels quite long. A user would have to do `max_a_posteriori.optimized_fitness()`, wonder if it's worth shortening to `fitness()` and `value()`?

Reply:
I changed it to `getFitness()` and `getValueFor(...)`